### PR TITLE
Gmap

### PR DESCRIPTION
**Pull request by cassiemak**
gmap styling
marker icon
awesome font stars - ng-bind-html
filter toggle done

### CODE REVIEW

File: app/assets/javascripts/angular/controllers/home_controller.js.erb [hunk 1]
```diff
@@ -24,32 +27,103 @@ app.controller('HomeCtrl', ['$scope', '$http', '$routeParams', 'uiGmapGoogleMapA
...
+      if (i == 0.5) {
+        stars = stars + "<i class='icon fa fa-star-outline'></i>";
+      }
+    }
+    // must trust html for it to show up
+    return $sce.trustAsHtml(stars);
+  }
+
+  // styling options, array
+  var hike = [
```
Comment (paulld) [inline app/assets/javascripts/angular/controllers/home_controller.js.erb]: can you think of a better name for `hike`? those are the map styling options...

File: app/assets/templates/pages/home.html [hunk 2]
```diff
@@ -36,81 +36,97 @@
...
-    
-    <div class="one-trail" ng-show="trail">
-      <div>
-        <button ng-click="switchPanel()" id="back-button" class="btn glyphicon glyphicon-chevron-left"></button>
-        <h1 class="title-name">Trail Name</h1>
+    </div>    
+    <div id="one-trail" ng-show="trailPanel">
+      <div class="list-group-item" ng-click="switchPanel()">
+        <button id="back-button" class="btn glyphicon glyphicon-chevron-left"></button>
+        <h2 class="title-name">{{trail.name}}</h2>
```
Comment (paulld) [inline app/assets/templates/pages/home.html:83]: What is `trail` here?

Comment (paulld) [inline app/assets/templates/pages/home.html:83]: it looks like you're outside of the ng-repeat, so `trail` wouldn't be available. In the other hand `$scope.trail` (very dangerous to name it like that btw) is a boolean, so I don't get it

**Comment by paulld** [comment]
Please rebase

File: app/assets/templates/pages/home.html (hunk 2 above)
Comment (cassiemak) [inline app/assets/templates/pages/home.html:83]: ng-show=trail is deleted here
and changed to ng-show=trailPanel

basically it is a switch

File: app/assets/javascripts/angular/controllers/home_controller.js.erb (hunk 1 above)
Comment (cassiemak) [inline app/assets/javascripts/angular/controllers/home_controller.js.erb]: Done

**Pull request merged**